drivers/adc/ad9656/ad9656.c

```diff
@@ -116,21 +116,22 @@ int32_t ad9656_spi_write(struct ad9656_dev *dev,
  * 		   FAILURE otherwise
```

Comment:
Title of the commit message is a bit too long.

Reply:
Squashed the commit to the previous commits